config/samples/resources/computetargethttpsproxy/target-https-proxy-with-certificate-manager-certificates/compute_v1beta1_computetargethttpsproxy.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeTargetHTTPSProxy
+metadata:
+  name: computetargethttpsproxy-sample
```

Comment:
Similarly could you please update this to something like `computetargethttpsproxy-sample-certmgr` to ensure unique sample names?

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#sample-naming-convention

Reply:
Done.